### DISCUSSION TITLE
Soroban Quest 3 Video

### DISCUSSION THREAD

**Discussion by rawritude** [stellar/sorobanathon#55] [General]
Hey Stellar peeps!

In the following video I go over how to retrieve a Contract ID from the Stellar Lab using either the Transaction Meta or the Footprint and go over how to decode XDR within the CLI.

[-->Link to video](https://youtu.be/ZrmDQKU2dqY)
![image](https://user-images.githubusercontent.com/7675978/202879922-fb36fa3b-d537-496f-b31e-241792ec793e.png)

**Comment by achallagundla** [comment]
Heya @rawritude ,
Congratulations! Your contribution to Sorobanathon: First Light has been accepted! 🎉
Your submission may qualify for a reward of 3000 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions. (You can also message me on discord @ anuxhya)

All the best,
Anuhya

**Comment by achallagundla** [comment]
PS: No need to fill out the form twice, if you have already done it once you are set! 
In the case that you have a wallet address change please discord message me individually.
